src/tests/Interop/PInvoke/Miscellaneous/ThisCall/ThisCallTest.cs

```diff
@@ -1,12 +1,12 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Runtime.InteropServices;
 using System;
 using System.Reflection;
 using System.Text;
 using TestLibrary;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
```

Comment:
I would also like to see tests for when a `CLong`/`CULong`/`NFloat` are a field in a value type. The marshalling is interesting to validate. Additionally validating the behavior of `sizeof()` and `Marshal.SizeOf()` for both an instance as well as the impact on the value type when the type is a field.

Reply:
I've added additional tests for these cases.